src/app/detail/detail.controller.js

```diff
@@ -34,8 +34,12 @@ class JobDetailController {
         return this.ShareService.linkedin(job);
     }
 
-    emailLink(job, toEmail) {
-        return this.ShareService.emailLink(job, toEmail);
+    sendEmailLink() {
```

Comment:
@jdobler-bullhorn when inserting new members into a type, alphabatize them relative to other members.